### PR TITLE
Reconciling PumpEvent timestamps to Dates using the pump's UTC offset

### PR DESCRIPTION
**Pull request by loudnate**
- Removes all uses of `localTimeZone` and `defaultTimeZone` from history parsing, instead using `pumpState.timeZone`

Note that this is stacked on #120

### CODE REVIEW

File: RileyLink.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1356,34 +1366,34 @@
 				TargetAttributes = {
 					430D64CA1CB855AB00FCA750 = {
 						CreatedOnToolsVersion = 7.3;
-						DevelopmentTeam = UY678SP37Q;
+						DevelopmentTeam = 57NRR26737;
```
Comment (loudnate) [inline RileyLink.xcodeproj/project.pbxproj]: My bad, I'll revert these.

File: RileyLinkKit/PumpOpsSynchronous.swift [hunk 2]
```diff
@@ -325,39 +325,35 @@ class PumpOpsSynchronous {
...
         
         let pumpModel = try getPumpModel()
-        
-        var pageNum = 0
-        var events = [PumpEvent]()
-        while pageNum < 16 {
+
+        var events = [TimestampedHistoryEvent]()
+
+        pages: for pageNum in 0..<16 {
```
Comment (loudnate) [inline RileyLinkKit/PumpOpsSynchronous.swift:335]: Here's some fun Swift: labeled control flows

File: RileyLink.xcodeproj/project.pbxproj (hunk 1 above)
Comment (ps2) [inline RileyLink.xcodeproj/project.pbxproj]: I have them reverted locally; if you want, I can commit.

File: RileyLinkKit/PumpOpsSynchronous.swift (hunk 2 above)
Comment (ps2) [inline RileyLinkKit/PumpOpsSynchronous.swift:335]: now we're back to gotos. ;)

**Comment by ps2** [comment]
NS uploading of temps and boluses appears to be working great.

File: RileyLinkKit/PumpOpsSynchronous.swift (hunk 2 above)
Comment (loudnate) [inline RileyLinkKit/PumpOpsSynchronous.swift:335]: Hahahaha I suppose you're right! This was the only use case in 2 years I've found for them.

**Pull request merged**